src/clj_gatling/reporters/raw_reporter.clj

```diff
@@ -23,7 +23,7 @@
   (with-open [wtr (BufferedWriter. (FileWriter. file-name))]
     (loop [lines-left lines]
       (let [line (pr-str (first lines-left))]
-        (.write	wtr line)
+        (.write  wtr line)
```

Comment:
This formatting rule I don't understand. Why two spaces are needed here?

Reply:
They aren’t. I must have hit space accidentally and didn’t notice because it didn’t break the tests. Will fox.

Reply:
Done!